packages/common/container-definitions/src/runtime.ts

```diff
@@ -67,6 +67,8 @@ export interface IRuntime extends IDisposable {
 
 	/**
 	 * Processes the given op (message)
+	 * @param message - delta message received from the server
+	 * @param local - true if the message was originally generated by the current client
```

Comment:
"current" is not correct. It should be:
true if the message was originally generated by the client receiving it.